.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Release Binaries
+
+on: 
+  release:
+    types: [created]
+
+jobs:
+  releases-matrix:
+    name: Release Go Binary
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # build and publish in parallel: linux/386, linux/amd64, windows/386, windows/amd64, darwin/386, darwin/amd64 
+        goos: [linux, windows, darwin]
+        goarch: ["386", amd64]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: wangyoucao577/go-release-action@master
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        goos: ${{ matrix.goos }}
+        goarch: ${{ matrix.goarch }}
+        goversion: "https://dl.google.com/go/go1.14.6.linux-amd64.tar.gz"
```

Comment:
can this always be the latest Go? If not can you do something like
`"https://dl.google.com/go/go1.14.6.${{ matrix.goos }}-${{ matrix.goarch }}.tar.gz"`

Reply:
Don't think this action uses semver, could possibly swap it out for our own solution that uses `setup-go`

Reply:
Additionally using that info from the matrix is wrong since it's cross compiling on linux

Reply:
ahh I see